x/concentrated-liquidity/fees.go

```diff
@@ -58,13 +58,13 @@ func (k Keeper) chargeFee(ctx sdk.Context, poolId uint64, feeUpdate sdk.DecCoin)
 	return nil
 }
 
-// initializeFeeAccumulatorPosition initializes the pool fee accumulator with given liquidity delta and zero value for the accumulator.
+// initializeFeeAccumulatorPosition initializes the pool fee accumulator with zero liquidity delta
+// and zero value for the accumulator.
 // Returns nil on success. Returns error if:
 // - fails to get an accumulator for a given poold id
-// - attempts to re-initialize an existing non-zero liqudity position
+// - attempts to re-initialize an existing fee accumulator liqudity position
 // - fails to create a position
-// nolint: unused
-func (k Keeper) initializeFeeAccumulatorPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, lowerTick, upperTick int64, liquidityDelta sdk.Dec) error {
+func (k Keeper) initializeFeeAccumulatorPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, lowerTick, upperTick int64) error {
```

Comment:
Note: this change is made because we should never have to re-initialize an existing position.

When withdrawing all liquidity from a position, the rewards will be claimed and the accumulator position deleted while claiming